dts/bindings/timer/st,stm32-lptim.yaml

```diff
@@ -45,3 +45,10 @@ properties:
       - 32
       - 64
       - 128
+
+  st,counter-value:
```

Comment:
Not sure if the name is clear enough. We can ask around for more explicit names.

Reply:
what about `st,timeout`